### DISCUSSION TITLE
How to add a border to the overlay terminal?

### DISCUSSION THREAD

**Discussion by louwers** [LazyVim/LazyVim#1200] [Q&A]
The overlay terminal has the same color as the background for me.

I wonder if it is possible to add a border to it.

From my research it uses edgy & toggleterm: https://github.com/LazyVim/LazyVim/blob/142e6bec209704210db23b3424b4d51896bb206a/lua/lazyvim/plugins/extras/ui/edgy.lua#L21

I don't see any options for adding a border with edgy: https://github.com/folke/edgy.nvim

E.g. 

<img width="224" alt="image" src="https://github.com/LazyVim/LazyVim/assets/649392/ca47c4d3-85db-45d4-99db-c076f9e056b1">

**Comment by pkazmier** [comment]
You might try changing the border color in Tokyonight theme. I prefer more distinct separation in windows. Here is my tweak:
```lua
  {
    "folke/tokyonight.nvim",
    opts = {
      on_colors = function(colors)
        colors.border = colors.comment
      end,
    },
  },

```

And screenshots to show what it looks like:
<img width="1260" alt="image" src="https://github.com/LazyVim/LazyVim/assets/747855/e9d9ad9b-9319-4049-8850-1acd520fe7b2">
<img width="1260" alt="image" src="https://github.com/LazyVim/LazyVim/assets/747855/b5730dc0-35ff-46a7-bebb-bced6a6fc17f">

**Comment by FBorowiec** [comment]
Up I have the same issue

**Comment by jyuan0** [comment]
The overlay terminal is created by lazy.nvim. You can add a border by adding a key to your lazy.nvim spec (in `config/lazy.lua`):
```lua
  ui = {
    -- The border to use for the UI window. Accepts same border values as |nvim_open_win()|.
    border = "rounded",
  },
```
https://github.com/folke/lazy.nvim#%EF%B8%8F-configuration

Note: the `border` setting also applies to the lazy.nvim UI.

Edit: I'm assuming the "overlay" terminal is the one spawned with keymap `<C-/>` (Control-Slash).

**Comment by dimas-cyriaco** [comment]
I fixed this by remapping the key:

```lua
local Util = require("lazyvim.util")

vim.keymap.set("n", "<C-/>", function()
  Util.terminal(nil, { border = "rounded" })
end, { desc = "Term with border" })
```

More info here: https://github.com/LazyVim/LazyVim/discussions/1915
